.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,4 +9,5 @@
 
 ## Checklist
 - [ ] You followed [contributing section](https://github.com/dbt-athena/dbt-athena#contributing)
-- [ ] You tested your changes
+- [ ] You added unit tests if not possible and necessary
```

Comment:
I would rewrite this sentence to make it more clear.

Reply:
@mattiamatrix good catch, this was indeed not was I wanted to write, have a look again.